WordPress/WordPressTest/BlogJetpackTests.swift

```diff
@@ -151,6 +151,43 @@ class BlogJetpackTests: CoreDataTestCase {
         XCTAssertEqual(2, Blog.count(in: mainContext))
     }
 
+    /// Verify an account's blogs won't be saved if the account is deleted during a blog sync.
+    func testSyncBlogsAndSignOut() throws {
+        let wpComAccount = try createOrUpdateAccount(username: "user", authToken: "token")
+        XCTAssertEqual(Blog.count(in: mainContext), 0)
+        var deleted = false
+
+        // Blog sync makes a series of HTTP requests: the first one fetchs all blogs, followed by a few
+        // requests to get blog capabilities (one for each blog).
```

Comment:
IMO, it's not efficient to make individual HTTP requests to the same endpoint. I've created https://github.com/wordpress-mobile/WordPress-iOS/issues/20964 to suggest a potential improvement: creating a new API endpoint that accepts a list of site id and returns their Jetpack capablities.

Reply:
Worth adding a link to the issue, in case we'll get to work on it and do a search in the codebase for its reference?
```suggestion
        // requests to get blog capabilities (one for each blog).
        //
        // See also https://github.com/wordpress-mobile/WordPress-iOS/issues/20964
```